integrations/comscore/lib/index.js

```diff
@@ -5,6 +5,7 @@
  */
 
 var integration = require('@segment/analytics.js-integration');
+const { option } = require('@segment/analytics.js-integration/lib/statics');
```

Comment:
@pooyaj Elena made changes to our Comscore web integration and she noticed that this line was added. She didn't add it and it's not on master. Have you seen something like this get automatically added before? I'm not sure we need it so thinking we just remove it but wanted to double check!

Reply:
Let's remove this. Probably it is an auto-import by the IDE, but don't think it is needed here.

Reply:
hey @pooyaj @kdaswani the line is removed!